client_packages/chat/index.html

```diff
@@ -0,0 +1,18 @@
+<html>
+    <head>
+        <link rel="stylesheet" type="text/css" href="style/checkbox.css" media="screen">
+        <link rel="stylesheet" type="text/css" href="style/main.css" media="screen">
+        <meta charset="utf-8" />
+    </head>
+    <body>
+        <div id="chat" class="ui_element">
+            <ul id="chat_messages"></ul>
+        </div>
+        <script
+			src="https://code.jquery.com/jquery-3.3.1.min.js"
```

Comment:
move it to 11 line, like that:  `<script src="https://code.jquery.com/jquery-3.3.1.min.js"`